osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -21,6 +21,11 @@ public class CatcherArea : Container
         public readonly Catcher MovableCatcher;
         private readonly CatchComboDisplay comboDisplay;
 
+        /// <summary>
+        /// Whether <see cref="DrawablePalpableCatchHitObject"/> fruit should appear on the plate.
+        /// </summary>
+        public bool CatchFruitOnPlate { get; set; } = true;
```

Comment:
Was there intent behind not placing this in `Catcher` directly, which would save on passing this value on every new result (and the test changes)? On a quick check it seems like that would work just as well.

Reply:
Can move this down. The intention was just to propagate the mod setting one level less, but i *guess* `Catcher` is already publicly exposed so it doesn't mean much?